### PR TITLE
Register alphacoder.is-a.dev

### PR DESCRIPTION
**Pull request by XZeipher**
Register alphacoder.is-a.dev with CNAME record pointing to alphacoder.is-a.dev.

### CODE REVIEW

**Comment by XZeipher** [comment]
Hehe

File: domains/alphacoder.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "XZeipher",
+           "email": "rishavroy200@gmail.com",
+           "discord": "874154911212896267"
+        },
+    
+        "record": {
+            "CNAME": "alphacoder.is-a.dev"
```
Comment (CuteDog5695) [inline domains/alphacoder.json:9]: You can't point CNAME record to the subdomain you are trying to get.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> Hehe

?

**Comment by orxngc** [comment]
Closing as stale

**Pull request closed without merging**